python/ctsm/toolchain/gen_mksurfdata_jobscript_multi.py

```diff
@@ -12,41 +12,45 @@
 from ctsm.utils import abort
 
 valid_scenarios = [
+    "global-vic",
+    "global-potveg",
```

Comment:
I added potveg and vic here. 

My concerns about vic:
- I'm unclear whether we want 16pft files or the more versatile 78pft files.
- I think the --vic option does not change the name of the fsurdat file, does it? If not, then the Makefile will overwrite (?) the vic files with the non-vic files (or vice versa).

Reply:
VIC just adds some additional fields to the files. So the idea is that for the VIC resolutions we'll have those additional fields on the files that are used. So you just use the VIC version and don't worry about a non-VIC version. What we have to manage is what resolutions it'll be possible to run VIC for.

We might as well just have VIC for the 78pft files. I think we had the 16pft files previously because we had to have both the 16pft version and the crop version. Now, we should just support the 78 version so you can use in either case.

Reply:
Ah, this helps. In the spreadsheet we indicated f09, f19, f10 as our VIC resolutions. 

I'm resolving this by adding the --vic option to `global-present` and `global-present-low-res` so that VIC is on for a few additional resolutions. Based on your comment, this should not be a problem.